kitchen.yml

```diff
@@ -40,11 +40,6 @@ suites:
       name: terraform
       root_module_directory: test/fixtures/postgresql-public
       command_timeout: 1800
-  - name: postgresql-public-iam
```

Comment:
Why are we removing `postgresql-public-iam`, I still see a test for this

Reply:
There's an issue with the provider that was causing the test to fail. I've reported it here [#9862](https://github.com/hashicorp/terraform-provider-google/issues/9862). Once it goes live we can get this back in. Right now it's going to block the build.

Reply:
Thanks, let's open an issue in the repo to re add this back once upstream releases.

Reply:
https://github.com/terraform-google-modules/terraform-google-sql-db/issues/243